Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidGameController.cs

```diff
@@ -32,9 +32,8 @@ public unsafe struct AndroidGameControllerState : IInputStateTypeInfo
         [InputControl(name = "select", bit = (uint)AndroidKeyCode.ButtonSelect, variants = kVariantGamepad)]
         public fixed uint buttons[(kMaxAndroidButtons + 31) / 32];
 
-
-        [InputControl(name = "leftTrigger", offset = (uint)AndroidAxis.Ltrigger * sizeof(float) + kAxisOffset, variants = kVariantGamepad)]
-        [InputControl(name = "rightTrigger", offset = (uint)AndroidAxis.Rtrigger * sizeof(float) + kAxisOffset, variants = kVariantGamepad)]
+        [InputControl(name = "leftTrigger", offset = (uint)AndroidAxis.Brake * sizeof(float) + kAxisOffset, variants = kVariantGamepad)]
```

Comment:
Not sure why this is being shown as a change, because it matches the code in develop branch